### DISCUSSION TITLE
How can I import my existing docker-compose.yml?

### DISCUSSION THREAD

**Discussion by Milo123459** [louislam/dockge#249] [Ask for Help] [answered]
### ⚠️ Please verify that this bug has NOT been raised before.

- [X] I checked and didn't find similar issue

### 🛡️ Security Policy

- [X] I agree to have read this project [Security Policy](https://github.com/louislam/dockge/security/policy)

### 📝 Describe your problem

I have a docker-compose.yml, and I was wondering what the best way to migrate was? It uses volumes and environment variables with multiple ports being exposed.

*I saw a similar issue, but without an answer.*

### 🐻 Dockge Version

n/a

### 💻 Operating System and Arch

Raspbian Bullseye

### 🌐 Browser

n/a

### 🐋 Docker Version

20

### 🟩 NodeJS Version

20

**Comment by Zaid-maker** [comment, marked as answer]
Copy paste your existing `docker-compose.yml` in the editor when creating new stack.
